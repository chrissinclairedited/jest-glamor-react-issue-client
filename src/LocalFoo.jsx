import React from "react";
import { css } from "glamor";

export default () => <p {...css({fontWeight: 'bold'})}>Some locally styled component</p>;
