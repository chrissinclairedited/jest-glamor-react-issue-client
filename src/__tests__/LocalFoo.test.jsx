import React from "react";
import { mount } from "enzyme";
import LocalFoo from "../LocalFoo";

describe("<LocalFoo />", () => {
    it("works well for local components", () => {
        const component = mount(<LocalFoo />);
        expect(component).toMatchSnapshot();
    })
})
