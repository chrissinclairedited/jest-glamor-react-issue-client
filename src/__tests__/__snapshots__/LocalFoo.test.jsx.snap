// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocalFoo /> works well for local components 1`] = `
.css-0,
[data-css-0] {
  font-weight: bold;
}

<Component>
  <p
    data-css-0=""
  >
    Some locally styled component
  </p>
</Component>
`;
